site/compiler/index.js

```diff
@@ -86,8 +84,6 @@ export function compileRoutes(state) {
       jsPath,
       state: encodedState,
     });
-    const ejsMs = Date.now() - ejsStart;
-    console.log(`Compiled ${route.filePath} render=${renderMs} ejs=${ejsMs}`);
```

Comment:
Do we want to get rid of this? I thought it increased visibility of the compilation process so we can see what goes wrong if it fails in the publish lambda.